mod/wet4/views/default/wet4_theme/custom_css.php

```diff
@@ -1382,8 +1382,9 @@
 
 .custom-message{
     position: fixed;
-    width: 1140px;
-    z-index: 9999;
+    width: 95%;
+    max-width: 1140px;
+    z-index: 1039;
     top: 185px;
```

Comment:
can we modify the top a little so that it's not in the way of the site content? I know they want it to fade away but I don't know about the accessibility of that. Tabbing to the X works and clicking anywhere on the alert also dismisses it 👌